[2021-08-08 17:15:03,566] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: popula_bd.verifica_arquivo 2021-08-08T20:14:51.888156+00:00 [queued]>
[2021-08-08 17:15:03,568] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: popula_bd.verifica_arquivo 2021-08-08T20:14:51.888156+00:00 [queued]>
[2021-08-08 17:15:03,568] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-08-08 17:15:03,568] {taskinstance.py:1088} INFO - Starting attempt 1 of 2
[2021-08-08 17:15:03,568] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-08-08 17:15:03,572] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): verifica_arquivo> on 2021-08-08T20:14:51.888156+00:00
[2021-08-08 17:15:03,574] {standard_task_runner.py:52} INFO - Started process 24883 to run task
[2021-08-08 17:15:03,575] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'popula_bd', 'verifica_arquivo', '2021-08-08T20:14:51.888156+00:00', '--job-id', '229', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/popula.py', '--cfg-path', '/tmp/tmpgoip_oj1', '--error-file', '/tmp/tmp78zije6w']
[2021-08-08 17:15:03,575] {standard_task_runner.py:77} INFO - Job 229: Subtask verifica_arquivo
[2021-08-08 17:15:03,586] {logging_mixin.py:104} INFO - Running <TaskInstance: popula_bd.verifica_arquivo 2021-08-08T20:14:51.888156+00:00 [running]> on host DESKTOP-BEFA8CP.localdomain
[2021-08-08 17:15:03,600] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=vh15fleury@hotmail.com
AIRFLOW_CTX_DAG_OWNER=victor hugo
AIRFLOW_CTX_DAG_ID=popula_bd
AIRFLOW_CTX_TASK_ID=verifica_arquivo
AIRFLOW_CTX_EXECUTION_DATE=2021-08-08T20:14:51.888156+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-08T20:14:51.888156+00:00
[2021-08-08 17:15:03,606] {python.py:151} INFO - Done. Returned value was: None
[2021-08-08 17:15:03,607] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=popula_bd, task_id=verifica_arquivo, execution_date=20210808T201451, start_date=20210808T201503, end_date=20210808T201503
[2021-08-08 17:15:03,616] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-08-08 17:15:03,627] {local_task_job.py:149} INFO - Task exited with return code 0
